indigo_api/models/tasks.py

```diff
@@ -154,7 +155,8 @@ def decorate_potential_assignees(cls, tasks, country):
             if task.state == 'open':
                 task.potential_assignees = [u for u in potential_assignees if task.assigned_to_id != u.id]
             elif task.state == 'pending_review':
-                task.potential_assignees = [u for u in potential_reviewers if task.assigned_to_id != u.id and task.submitted_by_user_id != u.id]
+                task.potential_assignees = [u for u in potential_reviewers if task.assigned_to_id != u.id and
```

Comment:
I think `potential_reviewers` should include users with the `close_any_task` perm as well. We can't assume that if you have `close_any_task` you also have `close_task`.

So line 152 is `... if u.has_perm('indigo_api.close_task') or u.has_perm('indigo_api.close_any_task')`